executor/executor.go

```diff
@@ -21,7 +23,10 @@ func (e ErrStatus) Error() string {
 }
 
 // Run executes a slice of CommandDefs
-func Run(output io.Writer, job screwdriver.JobDef) error {
+func Run(path string, output io.Writer, job screwdriver.JobDef) error {
+	if err := chdir(path); err != nil {
```

Comment:
Why are we doing this instead of just passing it as the working directory for execute?(os/exec.cmd.Dir)

Like this: https://github.com/screwdriver-cd/launcher/blob/master/git/git.go#L111